prboom2/src/r_draw.c

```diff
@@ -587,3 +591,14 @@ int R_GetFuzzPos()
 {
   return fuzzpos;
 }
+
+void R_ResetFuzzCol(int rows)
+{
+  R_ResetColumnBuffer(); // Flush current columns before changing fuzzpos
+  fuzzpos = (fuzzpos + (rows / fuzzcellsize)) % FUZZTABLE;
+}
+
+void R_NewFuzzCol(int x, int rows)
+{
+  if (!(x % fuzzcellsize)) R_ResetFuzzCol(rows);
```

Comment:
It's a bit confusing in this code base because there are many styles, but please take at least 2 lines for an `if`.